service/src/main/java/bio/terra/profile/service/profile/ProfileService.java

```diff
@@ -233,7 +238,15 @@ private Organization getProfileOrganization(BillingProfile profile) {
             profile
                 .subscriptionId()
                 .map(enterpriseConfiguration.subscriptions()::contains)
-                .orElse(false));
+                .orElse(false))
+        .limits(
+            profile
+                .subscriptionId()
+                .flatMap(
```

Comment:
I don't think `profile.subscriptionId()` is correct - this would look up the subscriptionId in the terra-helmfile config. Instead, I believe we want to be looking up the billing profile id, which would be `profile.id()`.

In other words, we are applying limits to specific billing profiles, not to entire subscriptions.

Maybe this means "limits" should be moved out from under "organization" in the response? Hmmm …

Reply:
Do we need the extra granularity? Or is using Organization / subscription sufficient for AnVIL Lite and trial accounts?